saos-restful-web-api/src/main/java/pl/edu/icm/saos/api/dump/judgment/views/DumpJudgmentsView.java

```diff
@@ -106,6 +120,8 @@ public String toString() {
                     .add("sinceModificationDate", sinceModificationDate)
                     .toString();
         }
+
```

Comment:
withGenerated flag should be included in toString() hashCode() and equals() methods